dataproc/quickstart/quickstart.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This quickstart shows how you can use the Cloud Dataproc Client library to create a
+// Cloud Dataproc cluster, submit a PySpark job to the cluster, wait for the job to finish
+// and finally delete the cluster.
+
+// [START dataproc_quickstart]
```

Comment:
```
// [START dataproc_quickstart]

// The quickstart command shows how you can...
package main
```

Nice to have the comment in the sample. The newline between the region tag and the comment makes it so the region tag doesn't show up in the GoDoc for the command.